### PR TITLE
Javascript-pizzeria-lauralyckholm

### PR DESCRIPTION
**Pull request by LauraLyckholm**
View live here: https://javascript-pizzeria-lauralyckholm.netlify.app/

The assignment was to create a pizzeria handling orders simply by using Javascript, to excercise the use of variables, conditionals etc.

I was first a little confused regarding how the prompt would be saved. I first wrote it exactly as alert(), but then quickly realized I needed to add const or let before it in order to save the value that the customer writes into the prompt. I tested this, and basically everything using console.log()

I wanted my choices to be presented in new lines in the prompt-window, and found through google that \n works for this.
When doing the if/else-statement, I ran into a problem with "===" comparing exactly what was typed into the prompt-window = which was a string not a number. In some cases I changed for example 'xxx === 3' to 'xxx == 3', I also tried changing 3 for "3", and later I made a new variable using the method parseInt. In the future I will choose one way to do this and stick to it, but here I wanted to utilize different ways to do it, and also show you guys I know different ways to solve this issue.

It was really tricky to learn how to make switch cases, and especially nested switch cases. I googled a lot, and also the class we had with Diego on the first Tuesday helped a little. I had an issue with the last switch case always going to the default value. I solved this by replacing the variable created for the prompt with true, a solution I found googling.

I tried a little bit to stop the code from keeping to run after a final else or default alert, but if was tricky to get going. Since I felt the need to move on to other tasks I left it for now.

I decided to write my code without ";" after each line in this task, and tried to follow the "be consequent" rule. However, after speaking to my personal code coach at my current work place (where I will probably start working in the future) he told med that the ALWAYS write using ";", **so in the next project I plan on doing it that way.**

### CODE REVIEW

File: code/switch.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  case "1":
+    chosenFood = "Pizza"
+    alert(
+      `You've chosen ${chosenFood}!`
+    )
+    pizzaType = prompt(
+      `Delicious! What kind of ${chosenFood} do you want? Type a number.
+    \n1. Quattro Formaggio
+    \n2. Margherita
+    \n3. Swedish Kebab`
```
Comment (jennylarsen) [inline code/switch.js:45]: Have not seen this before but it looks nice, as I understand it has the function of creating spaces between the alternatives but correct me if I'm wrong?

File: code/switch.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    }
+    break
+  default: alert(
+    `I'm sorry, we don't have that on the menu.`
+  )
+}
+
+// Step 4 - Age
+// Step 5 - Order confirmation
+// I decided to nest the switch cases here aswell, so stept 4-5 are in the same section
```
Comment (jennylarsen) [inline code/switch.js:135]: Great idea to put step 4 and 5 together! Looks clean and does not interfere with the understanding of the code.

File: code/script.js [hunk 3]
```diff
@@ -1,19 +1,185 @@
...
+}
 
 // Step 3 - Subtype choice
-// Your code goes here
+// subtypeChoice defines a variable for the different subchoices. pizzaType, pastaType and saladType save the prompt values for display in the alert-message. Created in a nested if/else statement. 
+
+let pizzaType
+let pastaType
+let saladType
+let subtypeChoice
```
Comment (jennylarsen) [inline code/script.js:56]: Good naming, makes it easy to understand the code!

**Review by jennylarsen — COMMENTED** [review]
GREAT JOB Laura! Super proud! 
When trying out the application it works as it is supposed to, both when following the instructions and also when typing invalid input, nice job!
The code fulfills the requirements and the stretch goals in my opinion. I think that the code is well described with comments but also with understandable variables.

Something I noticed and wonderd about but that I saw you also commeting on, is not using ;
But as I understand in the comment you will start using this in the future, so that is good. I have not marked it in the code since it is not used at all but this comment is my over all note! You should be really happy with this project, hard to find anything to comment about! <3

File: code/switch.js (hunk 1 above)
Comment (LauraLyckholm) [reply code/switch.js:45]: Yes, exactly! I think I wrote about it in my readme, but I might have deleted it.. It makes for breaks in the visual appearance ☺️

**Pull request closed without merging**